modules/docs/pages/replication-protocol.adoc

```diff
@@ -374,9 +374,11 @@ The recipient MUST send a response unless the request was sent
 added the revision to its database, or has failed to add it. On success
 the response can be empty; on failure it MUST be an error.
 
-Note: The recipient may need to send one or more `getattach` messages
+The sender may send this as `noreply` if there are no attachments or `deltaSrc` is not present.
+
+Note: The recipient may need to send one or more `getAttachment` messages
 while processing the `rev` message, in which case it MUST NOT send the
-`rev`'s response until it's received responses to the `getattach`
+``rev``'s response until it's received responses to the `getAttachment`
```

Comment:
Why the need for double backtick?

Reply:
Otherwise, this whole section is indented.